extra_docs/realtime/channels.md

```diff
@@ -0,0 +1,73 @@
+## Communication with channels
+
+Real Time Module supports communications using channels. You can create channels with  [Jexia Web Management Application](https://docs.jexia.com/getting-started/user-management/).  
+It is possible to send messages to the channel and everyone subscribed to that channel will immediately receive them.   
+Channel is also able to keep messaging history.  
+  
+### Initialize  
+To use real time communication initialize Jexia client with the real time module:  
+  
+``` javascript
+import { jexiaClient, realTime } from "jexia-sdk-js";  
+  
+const rtm = realTime();  
+  
+jexiaClient().init({  
+ projectID: "your-project-id",  key: "your-api-key",    
+  secret: "your-api-secret"  
+}, rtm);  
+```  
+
+After Jexia client has been initialized, you can obtain your channel object just by name:
+
+```javascript
+const channel = rtm.channel('my_channel');
+```
+
+  
+### Subscribe to the channel events
+Basically, channel object is inherited from the [RxJS Subject](https://rxjs-dev.firebaseapp.com/api/index/class/Subject), so just use `subscribe()` method: 
+  
+```javascript
+channel.subscribe(message => {
+  // you received a message here!
+});
+```
+
+You can use `.pipe()` and any RxJS operators to have more flexibility:
+
+```javascript
+channel.pipe(
+  filter(message => message.from === user.id), // filter only specific user
+  pluck('data') // pull out only data field 
+).subscribe(...);
```

Comment:
Do I need to put here the same subscribe(message => {});  Can you extend example here?